src/System.Private.ServiceModel/tests/Scenarios/SelfHostWcfService/ConfigBackup.xml

```diff
@@ -0,0 +1,176 @@
+  contract="WcfService.IWcfService" />
+
+        <!-- Kerberos and NTLM authentication -->
+        <!--
+        <endpoint address="https-ntlm" binding="basicHttpsBinding" bindingConfiguration="https+ntlm"
+          name="https-ntlm" contract="WcfService.IWcfService" />
+        <endpoint address="https-windows" binding="basicHttpsBinding"
+          bindingConfiguration="https+windows" name="https-windows" contract="WcfService.IWcfService" />
+-->
+        <!-- net.tcp endpoint addresses -->
+
+        <endpoint address="tcp-default" binding="netTcpBinding" bindingConfiguration="nettcpBinding"
+                  name="nettcp-defaultBinding" contract="WcfService.IWcfService" />
+        <endpoint address="tcp-nosecurity" binding="netTcpBinding" bindingConfiguration="nettcpBinding+nosecurity"
+                  name="nettcp-defaultBinding" contract="WcfService.IWcfService" />
+        <endpoint address="tcp-custombinding-nosecurity-text" binding="customBinding" bindingConfiguration="tcp+text"
+                  name="nettcp-customBinding" contract="WcfService.IWcfService" />
+
+        <!-- Metadata endpoint addresses -->
+        <!--<endpoint name="http-mex" address="mex" binding="mexHttpBinding" contract="IMetadataExchange"/>
+                <endpoint name="tcp-mex" address="mex" binding="mexTcpBinding" contract="IMetadataExchange"/>-->
+
+        <host>
+          <!--This part is important to determine the base addresses for your endpoints-->
+          <baseAddresses>
+            <add baseAddress="net.tcp://*:809/WindowsCommunicationFoundation" />
+            <add baseAddress="http://*:8081/WindowsCommunicationFoundation" />
+            <add baseAddress="https://*:44285/WindowsCommunicationFoundation" />
+          </baseAddresses>
+        </host>
+      </service>
+    </services>
+    <behaviors>
+      <serviceBehaviors>
+        <behavior>
+          <!-- To avoid disclosing metadata information, set the values below to false before deployment -->
+          <!--<serviceMetadata httpGetEnabled="true" httpsGetEnabled="true" />-->
+          <!-- To receive exception details in faults for debugging purposes, set the value below to true.  Set to false before deployment to avoid disclosing exception information -->
+          <serviceDebug includeExceptionDetailInFaults="true" />
+        </behavior>
+      </serviceBehaviors>
+    </behaviors>
+    <protocolMapping>
+      <add binding="basicHttpBinding" scheme="http" />
+    </protocolMapping>
+  </system.serviceModel>
+  <system.diagnostics>
+    <sources>
+      <source name="System.ServiceModel"
+        switchValue="Information, ActivityTracing"
+        propagateActivity="true">
+        <listeners>
+          <add name="traceListener"
+            type="System.Diagnostics.XmlWriterTraceListener"
+            initializeData="ServerTraces.svclog" />
+        </listeners>
+      </source>
+    </sources>
+  </system.diagnostics>
+</configuration>
```

Comment:
No. I'm just keeping it there as a reference since I haven't converted everything from config into code.